README.md

```diff
@@ -1041,6 +1043,7 @@ The snippet above will return all documents, which `telephones` array field cont
 | `$gte`       | `{a: {$gte: 10}}`               | Fields value is greater than or equal to the specified number.
 | `$exists`    | `{a: {$exists: true}}`          | Match if the field is present (or not if set to `false`) in the item, event if `nil`.
 | `$regex`     | `{a: {$regex: "fo[o]{1}"}}`     | Match regular expression on a field's value.
+| `$not`       | `{a: {$not: "fo[o]{1}"}}`       | Opposite of `$regex`.
```

Comment:
To get the exact functionality you are after, you would need to do:
`{a: {$not: {$regex: "fo[o]{1}"}}}`

Reply:
Ah.. I now see that `$not: string` is allowed as a short-hand for `$not: {$regex: string}` . I suppose that's fair enough.

Reply:
`$not: {$regex: string}` was added in Mongo 4.0.6 so I suppose `$not: string` is the conventional way. Regardless, if we decide to add support for `$not` as a general negation operator, both will be supported.